Makefile

```diff
@@ -576,18 +579,18 @@ govendor:
 .PHONY: fmt
 fmt: govendor genlocalfiles
 	@echo "+ $@"
-	@test -z "$$($(GOVENDOR) fmt +local)" || \
-		(echo "+ please format Go code with 'gofmt -s'" && /bin/false)
+	$(GOVENDOR) fmt +local || \
+		(echo "+ please format Go code with '$(GOFMT)'" && /bin/false)
 
 .PHONY: vet
 vet: govendor
 	@echo "+ $@"
-	test -z "$$($(GOVENDOR) tool vet $$( \
+	test -z "$$($(GOVENDOR) vet $$( \
 			$(GOVENDOR) list -no-status +local \
-			| perl -pe 's|$(SKYDIVE_GITHUB)/?||g' \
-			| grep -v '^tests') 2>&1 \
+			| grep -v '^$(SKYDIVE_GITHUB)/tests' \
+			| grep -v '^$(SKYDIVE_GITHUB)/contrib/collectd' \
+			) 2>&1 \
```

Comment:
what about '^tests' ?

Reply:
'^$(SKYDIVE_GITHUB)/tests' as I don't remove the prefix anymore